backend/Dockerfile

```diff
@@ -11,11 +11,6 @@ ENV APP_MODULE "backend.entrypoint:app"
 # Set WORKDIR
 WORKDIR /app
 
-RUN apt-get update -y && \
-    apt-get install -y --no-install-recommends ssh && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
-
 # add out prestart script launched by upstream's /start.sh and remove upstream entry
```

Comment:
```dockerfile
COPY pyproject.toml README.md tasks.py prestart.sh Pipfile Pipfile.lock src /app/
RUN \
    # prestart script launched by upstream's /start.sh and remove upstream entrypoint
    chmod +x /app/prestart.sh && rm -f /app/main.py && \
    pipenv sync --system
WORKDIR /app/src
```

Reply:
Actually I think we should not use his image.
There's not much value we loose some details and control.

Also, we'll deploy to k8s so his advise to run a single process makes sense.

```dockerfile
FROM python:3.11-slim

COPY pyproject.toml README.md tasks.py entrypoint.sh Pipfile Pipfile.lock /app/
RUN pip install --no-cache-dir pip pipenv && \
    chmod +x /app/entrypoint.sh && \
    cd /app && \
    pipenv sync --system
COPY src /app/src
WORKDIR /app/src

ENTRYPOINT ["/app/entrypoint.sh"]
CMD ["uvicorn", "backend.entrypoint:app", "--host", "0.0.0.0", "--port", "80"]
```

We'd have to turn the prestart into an entrypoint (adding `exec "$@"` at the end)

Reply:
> Also, we'll deploy to k8s so his advise to run a single process makes sense.

oh, we really don't need this image then.